alica_engine/include/engine/blackboard/Blackboard.h

```diff
@@ -42,6 +42,19 @@ class BlackboardImpl
     bool empty() const { return vals.empty(); }
     size_t size() const { return vals.size(); }
     std::unordered_map<std::string, std::any> vals;
+
+    std::unordered_map<std::string /*libraryname*/, std::any> _worldModels;
+
+    template <class T>
+    T* getWorldModel(const std::string& libraryName)
```

Comment:
- can't we use the existing `get` method? The key will be "worldModel" & type will be std::any for now.
- I don't think the library name should be necessary as a parameter. When namespacing is implemented (which can be done in a different PR) a `namespaced blackboard accessor` can internally deduce which namespace should be used.